website/data-storage/src/message/message.service.ts

```diff
@@ -66,13 +58,24 @@ export class MessageService {
   }
 
   public async setDispatchedStatus({ statuses, genesis }: IMessagesDispatchedKafkaValue): Promise<void> {
+    await sleep(1000);
     for (const messageId of Object.keys(statuses)) {
-      if (statuses[messageId] === 'Success') continue;
+      try {
+        await this.messageRepository.updateMessagePanicStatus(
```

Comment:
I think it will be better to use `messageRepository` methods that we already have. 

```javascript
        const messageToUpdate = this.messageRepository.getByIdAndGenesis(messageId, genesis)
        const status = statuses[messageId] === 'Success' ? false : true;
        messageToUpdate.processedWithPanic = status;
        this.messageRepository.save(messageToUpdate)
        ```